corehq/apps/data_cleaning/models.py

```diff
@@ -257,3 +259,28 @@ class BulkEditChange(models.Model):
 
     class Meta:
         ordering = ["created_on"]
+
+    def edited_value(self, case):
+        simple_transformations = {
+            EditActionType.REPLACE: lambda x: self.replace_string,
+            EditActionType.FIND_REPLACE: lambda x: x.replace(self.find_string, self.replace_string),
+            EditActionType.STRIP: str.strip,
```

Comment:
does strip actually handle ALL the whitespace characters including newlines, tabs, etc?
in the prototype I had to use a regex to properly do this, not just `strip`
https://github.com/dimagi/commcare-hq/blob/016784960037d260bcc7ca66eb3ed06f42bd4b24/corehq/apps/prototype/forms/data_cleaning.py#L349-L360

Reply:
nm looks like it does

Reply:
The docs say it does. It's working fine with tabs and newlines. Do you remember what use case was a problem in the prototype?